holidays/countries/bhutan.py

```diff
@@ -0,0 +1,230 @@
+        * [ROYAL GOVERNMENT OF BHUTAN 2015-16](https://web.archive.org/web/20151004161522/http://www.mohca.gov.bt/Publications/calander_2015-16.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2016-17](https://web.archive.org/web/20170606131500/http://www.mohca.gov.bt/download/calander_2016-17.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2017-18](https://web.archive.org/web/20180107115733/http://www.mohca.gov.bt/wp-content/uploads/2017/11/calander_2017-18.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2017-2018](https://web.archive.org/web/20180906031327/http://www.mohca.gov.bt/wp-content/uploads/2017/11/scan0126.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2019-2020](https://web.archive.org/web/20191221201614/http://www.mohca.gov.bt/downloads/scan0004.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2020-2021](https://web.archive.org/web/20200720030220/http://www.mohca.gov.bt/wp-content/uploads/2019/12/2020-Holiday-List0001.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2021-2022](https://web.archive.org/web/20230511011459/http://www.mohca.gov.bt/downloads/Holidays2021.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2022-2023](https://web.archive.org/web/20220619061040/https://www.mohca.gov.bt/wp-content/uploads/2021/11/HolidayList.pdf)
+        * [ROYAL GOVERNMENT OF BHUTAN 2023](https://web.archive.org/web/20221210152039/https://www.mohca.gov.bt/wp-content/uploads/2022/11/Holidays-2023.pdf)
+        * [Ministry of Home and Cultural Affairs ](https://web.archive.org/web/20100127021618/http://www.mohca.gov.bt/?mode=READMORE&news_id=157)
+    """
+
+    country = "BT"
+    # Jigme Khesar Namgyel Wangchuck ascended to the throne on December 9th, 2006.
+    start_year = 2007
+    subdivisions = (
+        "11",  # Paro
+        "12",  # Chhukha
+        "13",  # Haa
+        "14",  # Samtse
+        "15",  # Thimphu
+        "21",  # Tsirang
+        "22",  # Dagana
+        "23",  # Punakha
+        "24",  # Wangdue Phodrang
+        "31",  # Sarpang
+        "32",  # Trongsa
+        "33",  # Bumthang
+        "34",  # Zhemgang
+        "41",  # Trashigang
+        "42",  # Monggar
+        "43",  # Pemagatshel
+        "44",  # Lhuentse
+        "45",  # Samdrup Jongkhar
+        "GA",  # Gasa
+        "TY",  # Trashiyangtse
+    )
+
+    subdivisions_aliases = {
+        "Paro": "11",
+        "Chhukha": "12",
+        "Haa": "13",
+        "Samtse": "14",
+        "Thimphu": "15",
+        "Tsirang": "21",
+        "Dagana": "22",
+        "Punakha": "23",
+        "Wangdue Phodrang": "24",
+        "Sarpang": "31",
+        "Trongsa": "32",
+        "Bumthang": "33",
+        "Zhemgang": "34",
+        "Trashigang": "41",
+        "Monggar": "42",
+        "Pemagatshel": "43",
+        "Lhuentse": "44",
+        "Samdrup Jongkhar": "45",
+        "Gasa": "GA",
+        "Trashiyangtse": "TY",
+    }
+
+    def __init__(self, *args, **kwargs):
+        TibetanCalendarHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=BhutanHinduHolidays)
```

Comment:
```suggestion
        HinduCalendarHolidays.__init__(self, cls=BhutanHinduHolidays, show_estimated=True)
        TibetanCalendarHolidays.__init__(self)
```
Also, we need to define `estimated_label`.